src/main/java/cubicchunks/api/CubeGeneratorsRegistry.java

```diff
@@ -0,0 +1,101 @@
+/*
+ *  This file is part of Cubic Chunks Mod, licensed under the MIT License (MIT).
+ *
+ *  Copyright (c) 2015 contributors
+ *
+ *  Permission is hereby granted, free of charge, to any person obtaining a copy
+ *  of this software and associated documentation files (the "Software"), to deal
+ *  in the Software without restriction, including without limitation the rights
+ *  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ *  copies of the Software, and to permit persons to whom the Software is
+ *  furnished to do so, subject to the following conditions:
+ *
+ *  The above copyright notice and this permission notice shall be included in
+ *  all copies or substantial portions of the Software.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ *  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ *  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ *  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ *  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ *  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ *  THE SOFTWARE.
+ */
+package cubicchunks.api;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Random;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.google.common.primitives.Ints;
+
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+// Based on net.minecraftforge.fml.common.registry.GameRegistry.
+public class CubeGeneratorsRegistry {
+    private static Set<ICubicWorldGenerator> worldGenerators = Sets.newHashSet();
+    private static Map<ICubicWorldGenerator, Integer> worldGeneratorIndex = Maps.newHashMap();
+    private static List<ICubicWorldGenerator> sortedGeneratorList;
+
+    /**
+     * Register a cube generator - something that will change content of generated cube.
+     *
+     * @param generator           the generator
+     * @param modGenerationWeight a weight to assign to this generator. Heavy weights tend to sink to the bottom of
+     *                            list of world generators (i.e. they run later)
+     */
+    public static void registerWorldGenerator(ICubicWorldGenerator generator, int modGenerationWeight)
+    {
+        worldGenerators.add(generator);
+        worldGeneratorIndex.put(generator, modGenerationWeight);
+        if (sortedGeneratorList != null)
+        {
+            sortedGeneratorList = null;
```

Comment:
This doesn't look like the proper code format. The opening brackets should be inline with the if/else statements.